src/MainScene.tsx

```diff
@@ -63,17 +66,34 @@ export function MainScene(): JSX.Element {
     }
   }, [usdToCoinRates])
 
+  const handleOptionClick = (option): void => {
+    setCoinSelection(option)
+    setShowCodeScreen(true)
+  }
+
   return (
     <>
       <Header />
-      <h1>Edge Snack Bar</h1>
-      {Object.keys(currencyOptions)
-        .sort((a, b) => a.localeCompare(b))
-        .map(option => (
-          <p key={option}>
-            {option}: {usdToCoinRates[option]}
-          </p>
-        ))}
+      {showCodeScreen && (
+        <CodeScreen
+          coinSelection={coinSelection}
+          setShowCodeScreen={setShowCodeScreen}
+        />
+      )}
+      {!showCodeScreen && (
+        <>
+          {Object.keys(currencyOptions)
+            .sort((a, b) => a.localeCompare(b))
+            .map(option => (
+              <React.Fragment key={option}>
+                <button onClick={() => handleOptionClick(option)}>
+                  {option}: {usdToCoinRates[option]}
+                </button>
+                <br />
+              </React.Fragment>
+            ))}
+        </>
```

Comment:
This part can be separated into its own component.

Reply:
This will be addressed in a future PR: https://github.com/EdgeApp/edge-snack-bar-app/pull/6.